src/coreclr/nativeaot/Runtime/unix/UnixNativeCodeManager.cpp

```diff
@@ -566,7 +566,7 @@ int UnixNativeCodeManager::IsInProlog(MethodInfo * pMethodInfo, PTR_VOID pvAddre
 
     return savedFpLr && establishedFp ? 0 : 1;
 
-#elif defined(TARGET_ARM)
+#else // TARGET_ARM
```

Comment:
```suggestion
#elif defined(TARGET_ARM)
```
I would keep this as is and add `#else` with `#error Unsupported architecture` instead.

Reply:
Thanks, updated.